### PR TITLE
Reference trust anchor store in relying party section

### LINKED ISSUE #299: Introduction of trust anchor store in Layered Attestation (3.4) section
Does “trust anchor store” in Layered Attestation Environments require an introduction?

### PR DESCRIPTION
**Pull request by dthaler**
Also add a brief security considerations section, based on text from the TEEP architecture draft.

Fixes #299

Signed-off-by: Dave Thaler <dthaler@ntdev.microsoft.com>

### CODE REVIEW

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -1432,6 +1434,15 @@ the meantime the Attester has been compromised.
...
 
+## Trust Anchor Protection
+
+As noted in {{trustmodel}}, Verifiers and Relying Parties have trust anchor stores
+that must be secured.  Specifically, a Trust Anchor Store must resist
+modification against unauthorized insertion, deletion, and modification.
+
+If certificates are used as trust anchors, Verifiers and Relying Parties are also
+responsible for validating the entire certificate chain up to the trust anchor,
+which includes checking for certificate revocation.
```
Comment (thomas-fossati) [inline draft-ietf-rats-architecture.md:1445]: maybe we could reference the RFC(s) that are normative for certificate chain validation here?

**Review by thomas-fossati — COMMENTED** [review]
there are some inconsistencies around the capitalisation of [tT]rust [aA]nchor [sS]tore

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -860,7 +860,17 @@ trusts a Verifier that can appraise the trustworthiness of
 information about an Attester.  Such trust might come by the Relying
 Party trusting the Verifier (or its public key) directly, or might
 come by trusting an entity (e.g., a Certificate Authority) that is
-in the Verifier's certificate chain.
+in the Verifier's certificate chain.  Such trust
```
Comment (henkbirkholz) [inline draft-ietf-rats-architecture.md:863]: From RFC 4949

   $ certificate chain
      (D) Synonym for "certification path". (See: trust chain.)

      Deprecated Term: IDOCs SHOULD NOT use this term; it duplicates the
      meaning of a standardized term. Instead, use "certification path".

**Review by henkbirkholz — CHANGES_REQUESTED** [review]

File: draft-ietf-rats-architecture.md (hunk 1 above)
Comment (thomas-fossati) [reply draft-ietf-rats-architecture.md:1445]: i.e., [Section 6 of RFC5280](https://tools.ietf.org/html/rfc5280#section-6)

File: draft-ietf-rats-architecture.md (hunk 2 above)
Comment (dthaler) [reply draft-ietf-rats-architecture.md]: Done

File: draft-ietf-rats-architecture.md (hunk 1 above)
Comment (dthaler) [reply draft-ietf-rats-architecture.md]: Done

**Review by henkbirkholz — APPROVED** [review]

**Review by mcr — APPROVED** [review]

**Pull request merged**